app/views/school/la_funded_places/order.html.erb

```diff
@@ -0,0 +1,112 @@
+<%- title = t('page_titles.school_order_devices') %>
+<%- content_for :title, title %>
+<%- content_for :before_content do %>
+  <%- school_breadcrumbs items: title, school: @school, user: impersonated_or_current_user %>
+<%- end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      How to order
+    </h1>
+
+    <%= render Computacenter::TechSourceMaintenanceBannerComponent.new %>
+    <%= render partial: 'shared/half_term_delivery_suspension' %>
+
+    <h2 class="govuk-heading-m">
+      These laptops are currently in stock:
+    </h2>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        Google Chromebooks
+      </li>
+      <li>
+        Standard Windows laptops
+      </li>
+      <li>
+        DfE Restricted Windows laptops — low stock, delivery may take longer than 5 working days
+      </li>
+    </ul>
+    <p class="govuk-body">
+      <%= govuk_link_to 'It’s important to check you’re ordering the best type of laptop for each pupil.', laptop_types_school_path(@school) %>
+    </p>
+
+    <h2 class="govuk-heading-m">Chromebooks</h2>
+
+    <% if @school.preorder_information.will_need_chromebooks? %>
+      <p class="govuk-body">
+        You’ve told us you’ll order Chromebooks –&nbsp;
+        <%= govuk_link_to funded_chromebooks_school_path(@school) do %>
+          Change <span class="govuk-visually-hidden">Chromebook preference</span>
+        <% end %>
+      </p>
+      <div class="govuk-inset-text">
+        <p class="govuk-body">
+          After you place your order you’ll get an email asking for:
+        </p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>
+            the domain the school has registered for G Suite for Education, for example ‘school.co.uk’
+          </li>
+          <li>
+            a recovery email address. This should be on a separate domain to the one registered for G Suite for Education. For example, if the domain registered for G Suite for Education is ‘school.com’, the recovery email cannot be ‘recovery@school.com’
```

Comment:
... registered for Google Workspace for Education. For example ...

Reply:
Done